PseudoGantry/src/main.cpp

```diff
@@ -0,0 +1,143 @@
+#include <Arduino.h>
+#include "PseudoAxis.h"
+
+// pins and constants for testing pseudo gantry
+#define PWM_FREQ        245.
+#define PWM_PIN         11.
+#define STEPS_TO_TAKE   800.
+float delay_millis = ((1000/PWM_FREQ/2)*STEPS_TO_TAKE);
```

Comment:
I think the numbers all have to be floats to avoid integer truncation? ie `(1000.0/PWM_FREQ/2.0)*STEPS_TO_TAKE`

Reply:
I tested and wasn't getting any integer truncation, but I'll change them for consistency